pyzo/core/editorTabs.py

```diff
@@ -193,8 +193,9 @@ def pinned(self):
 class FindReplaceWidget(QtWidgets.QFrame):
     """ A widget to find and replace text. """
     
-    def __init__(self, *args):
-        QtWidgets.QFrame.__init__(self, *args)
+    def __init__(self, editors):
+        QtWidgets.QFrame.__init__(self, editors)
+        self.editors = editors
```

Comment:
It seems that the code that instantiates this widget has not been updated yet. Note that this bit might also not be necessary, as it can use the global `pyzo.editors` instead.

Reply:
I'm not sure what you mean about the code not being updated, I replaced *args because *args is only the parent window here. I didn't know about pyzo.editors, I can use it instead. Should I revert the change back to *args for __init__ ?

Reply:
I mean that the code that creates the widget should not pass `editors`, or does it already do this? If not, I'd recommend using `pyzo.editors`.